src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -990,7 +990,7 @@ def any(self, axis=0, bool_only=None, skipna=True, **kwargs):
                 return result.any(
                     axis=axis, bool_only=bool_only, skipna=skipna, **kwargs
                 )
-        return False if result is None else result
```

Comment:
The original implementation returns a value when result is None, but now it always returns result, is there a behavior change?

Reply:
Prior to #2002, it was not possible for the query compiler method to return None. In upstream Modin, it is also not possible for the result to be None, so this PR is going back to the behavior expected by the Modin frontend.